lib/ui/src/chat/CodeBlocks.tsx

```diff
@@ -97,6 +103,7 @@ export const CodeBlocks: React.FunctionComponent<CodeBlocksProps> = React.memo(f
     copyButtonClassName,
     insertButtonClassName,
     CopyButtonProps,
+    RecipesWidgetWrapper,
```

Comment:
Per the other suggestion,

```suggestion
    RecipesWidgetWrapper = null,
```